index.html

```diff
@@ -29,57 +31,56 @@ <h1 class="w3-display-topmiddle w3-container">Trick</h1>
     <!-- Host will switch to page where HOST can set up game with OPTIONS -->
     <!-- Guest player will be switching to a new page to enter the CODE from HOST  -->
     <main>
-        <section id='landing-page' class='page hidden'>
+        <section id='landing-page' class='page w3-display-container w3-center'>
             <h2>Welcome to Trick Trivia!</h2>
             <span id="player-name">
-                <label for="player-name-input">Player Name: </label>
-                <input id="player-name-input" type="text">
+                <label for="player-name-input">Player: </label>
+                <input id="player-name-input" type="text" placeholder=" Sign In">
             </span>
 
-            <div id="enter-game-options">
+            <div id="enter-game-options" class="w3-margin-top">
                 <button id='host-game-button'>Host Game</button>
                 <button id='join-game-button'>Join Game</button>
+                <button id="show-rule-hover" class="w3-margin-top" onclick="showRuleButton()">Rules</button>
```

Comment:
Could you remove the "onclick" rule here? We will do all of that with event listeners.

Reply:
Sure. onclick removed.

Reply:
Updated.